dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/PLL/PLL.mo

```diff
@@ -0,0 +1,125 @@
+within Dynawo.Electrical.Controls.PLL;
+
+model PLL "Phase-Locked Loop"
+
+  import Modelica.Blocks;
+  import Modelica.ComplexBlocks;
+  import Modelica.Constants;
+  import Modelica.SIunits;  
+    
+  import Dynawo.Connectors;
+
+  parameter SIunits.PerUnit Kp "PLL voltage calculation proportional gain";
+  parameter SIunits.PerUnit Ki "PLL voltage calculation integrator";
+  parameter SIunits.PerUnit fmin "Lower frequency limit (only positive values!)";
+  parameter SIunits.PerUnit fmax "Upper frequency limit";
+  parameter SIunits.PerUnit fref "Frequency refence value";
```

Comment:
save as above. Is fref supposed to be 1 all the time?

Reply:
yes, fref is always 1 in oder to limit the frequency deviation from 1p.u.
Should this be a protected parameter then or hard coded 1?

Reply:
It is fine like that!

Reply:
ok, fine. Thanks.